src/libraries/Microsoft.Extensions.FileProviders.Composite/src/CompositeFileProvider.cs

```diff
@@ -40,7 +40,7 @@ public CompositeFileProvider(IEnumerable<IFileProvider> fileProviders)
         /// Locates a file at the given path.
         /// </summary>
         /// <param name="subpath">The path that identifies the file. </param>
-        /// <returns>The file information. Caller must check Exists property. This will be the first existing <see cref="IFileInfo"/> returned by the provided <see cref="IFileProvider"/> or a not found <see cref="IFileInfo"/> if no existing files is found.</returns>
+        /// <returns>The file information. Caller must check Exists property. This will be the first existing <see cref="IFileInfo"/> returned by the provided <see cref="IFileProvider"/> or a not found <see cref="IFileInfo"/> if no existing files are found.</returns>
```

Comment:
```suggestion
        /// <returns>The file information. The caller must check the <c>Exists</c> property. This will be the first existing <see cref="IFileInfo"/> returned by the provided <see cref="IFileProvider"/> or a not found <see cref="IFileInfo"/> if no existing files are found.</returns>
```

Reply:
Or maybe just `<see cref="Exists" />`?

Reply:
```suggestion
        /// <returns>The file information. The caller must check the <see cref="Exists" /> property. This is the first existing <see cref="IFileInfo"/> returned by the provided <see cref="IFileProvider"/> or a not found <see cref="IFileInfo"/> if no existing files are found.</returns>
```